capa/features/extractors/dnfile/insn.py

```diff
@@ -57,68 +62,182 @@ def get_methods(ctx: Dict) -> Dict:
+                return
+
         # like System.IO.File::Delete
         yield API(str(callee)), ih.address
 
+    else:
+        # like kernel32.CreateFileA
+        for name in capa.features.extractors.helpers.generate_symbols(callee.module, callee.method):
+            yield API(name), ih.address
+
+
+def extract_insn_property_features(fh: FunctionHandle, bh, ih: InsnHandle) -> Iterator[Tuple[Feature, Address]]:
+    """parse instruction property features"""
+    insn: Instruction = ih.inner
+
+    prop_name: Optional[str] = None
+    prop_access: Optional[str] = None
+
+    if insn.opcode in (OpCodes.Call, OpCodes.Callvirt, OpCodes.Jmp, OpCodes.Calli):
+        if insn.operand.table == METHODDEF_TABLE:
+            """check if the method belongs to the MethodDef table and whether it is used to access a property"""
+            prop = get_properties(fh.ctx).get(insn.operand.value, None)
+            if prop is not None:
+                prop_name = str(prop)
+                prop_access = prop.access
+
+        elif insn.operand.table == MEMBERREF_TABLE:
+            """if the method belongs to the MemberRef table, we assume it is used to access a property"""
+            row: Any = resolve_dotnet_token(fh.ctx["pe"], insn.operand)
+            if row is None:
+                return
+            if not isinstance(row.Class.row, (dnfile.mdtable.TypeRefRow, dnfile.mdtable.TypeDefRow)):
+                return
+            if not row.Name.startswith(("get_", "set_")):
+                return
+
+            prop_name = DnType.format_name(
+                row.Class.row.TypeName, namespace=row.Class.row.TypeNamespace, member=row.Name[4:]
+            )
+            if row.Name.startswith("get_"):
+                prop_access = ACCESS_READ
+            elif row.Name.startswith("set_"):
+                prop_access = ACCESS_WRITE
+
+    elif insn.opcode in (OpCodes.Ldfld, OpCodes.Ldflda, OpCodes.Ldsfld, OpCodes.Ldsflda):
+        if insn.operand.table == FIELD_TABLE:
+            """determine whether the operand is a field by checking if it belongs to the Field table"""
+            read_field: Optional[DnType] = get_fields(fh.ctx).get(insn.operand.value, None)
+            if read_field:
+                prop_name = str(read_field)
+                prop_access = ACCESS_READ
+
+    elif insn.opcode in (OpCodes.Stfld, OpCodes.Stsfld):
+        if insn.operand.table == FIELD_TABLE:
+            """determine whether the operand is a field by checking if it belongs to the Field table"""
+            write_field: Optional[DnType] = get_fields(fh.ctx).get(insn.operand.value, None)
+            if write_field:
+                prop_name = str(write_field)
+                prop_access = ACCESS_WRITE
+
+    if prop_name is not None and prop_access is not None:
+        yield Property(prop_name, access=prop_access), ih.address
+        yield Property(prop_name), ih.address
```

Comment:
```suggestion

    yield Property(prop_name), ih.address
```

Reply:
yield this if `prop_access` is not set? check before that `prop_name` is set?

Reply:
addressed